dcc-import-variant/pom.xml

```diff
@@ -14,6 +14,12 @@
 
     <name>${project.artifactId}</name>
     <description>${project.name}</description>
+    <repositories>
+        <repository>
+        <id>osdc</id>
+        <url>https://nexus.osdc.io/repository/maven-releases</url>
+        </repository>
+    </repositories>
```

Comment:
just informational: we probably will not be able to merge this repo back to the original repo, since this it private and requires vpn